src/__init__.py

```diff
@@ -1,8 +1,24 @@
 from pkg_resources import get_distribution
+import codecs
+import json
 
 __version__ = get_distribution('rasa_nlu').version
 
 
 class Interpreter(object):
     def parse(self, text):
         raise NotImplementedError()
+
+    @staticmethod
+    def load_synonyms(entity_synonyms):
```

Comment:
actually here it's not a file...it's the dictionary itself. Why should we use `entity_synonyms_file`?